ui/src/main/resources/CKEditor/Translations.de.xml

```diff
@@ -90,22 +90,22 @@ ckeditor.plugin.save.failed=Der Inhalt kann nicht gespeichert werden aufgrund ei
 ckeditor.plugin.source.conversionFailed=Fehler beim Ausführen der Konvertierung.
 
 ### Missing: ckeditor.plugin.toolbar.basicStyles=Basic Styles
-### Missing: ckeditor.plugin.toolbar.lists=Lists
-### Missing: ckeditor.plugin.toolbar.bulletedlist=Bulleted List
-### Missing: ckeditor.plugin.toolbar.numberedlist=Numbered List
-### Missing: ckeditor.plugin.toolbar.insert=Insert
-### Missing: ckeditor.plugin.toolbar.horizontalrule=Horizontal Line
-### Missing: ckeditor.plugin.toolbar.specialchar=Special Character
-### Missing: ckeditor.plugin.toolbar.infoBox=Info Box
-### Missing: ckeditor.plugin.toolbar.successBox=Success Box
-### Missing: ckeditor.plugin.toolbar.warningBox=Warning Box
-### Missing: ckeditor.plugin.toolbar.errorBox=Error Box
-### Missing: ckeditor.plugin.toolbar.toc=Table of Contents
-### Missing: ckeditor.plugin.toolbar.include=Include
-### Missing: ckeditor.plugin.toolbar.code=Code Snippet
-### Missing: ckeditor.plugin.toolbar.otherMacros=Other Macros
-
-### Missing: ckeditor.inline.failedToInit=Failed to initialize CKEditor.
+ckeditor.plugin.toolbar.lists=Listen
+ckeditor.plugin.toolbar.bulletedlist=Liste mit Aufzählungszeichen
+ckeditor.plugin.toolbar.numberedlist=Nummerierte Liste
+ckeditor.plugin.toolbar.insert=Einfügen
+ckeditor.plugin.toolbar.horizontalrule=Horizontale Linie
+ckeditor.plugin.toolbar.specialchar=Sonderzeichen
+ckeditor.plugin.toolbar.infoBox=Infobox
+ckeditor.plugin.toolbar.successBox=Erfolg Box
+ckeditor.plugin.toolbar.warningBox=Warnbox
```

Comment:
Should there be a space between "Erfolg" and "box"? Same below. Thx

Reply:
@vmassol 
* "Infobox" exists in the dictionary https://context.reverso.net/translation/german-english/Infobox
* I found "Warnbox" defined as alert box inside a "Dictionary of Computing: Englisch-Deutsch (see "https://books.google.ro/books?id=9pOfBgAAQBAJ&printsec=frontcover#v=onepage&q&f=false).
For consistency we could use "Erfolgbox" but so far I did not find it as is on other references.

Reply:
Hello @oanat and @vmassol , 

- Infobox seems indeed to be defined in the [Duden](https://www.duden.de/). I'd recommend using this as authority and not a translation dictionary.
- Warnbox (oder Warnkasten) does not seem to exist but I see no mentioned alternative; Warnmeldung ("Warning message") is probably the closest; it is very common.
- Agreed on Erfolgbox instead of Erfolg box.

Reply:
I just wondered for a short moment if it should be "Erfolgsbox" instead of "Erfolgbox", but, nah.
I agree that the translations should all be a single word: "Infobox", "Warnbox", "Erfolgbox". as per the German language rule: "why use two words when a (slightly longer) single word suffices".

The alternatives like "Warnkasten", "Wanrmeldung" or "Warnhinweis" seem all to long for the drop down in the editor to me.